TyMSpa-2/front
# feature(#3) - Set up Redux Toolkit

**Changes**
- Installed dependencies for Redux Toolkit.
- Added a form on the user page that receives user data and dispatches an action to the corresponding store reducer upon submission.

**To test this:**
- Open a terminal and navigate to the project root.
- Run `yarn run dev`
- Go to the users page. Fill out the form and submit it. The new user will appear at the top of the list below.

**Checklist**
- [x] Commit messages follow the standars
- [x] I added reviewers
- [x] PR name follows the standards
- [x] I reviewed my PR
